CMakeLists.txt

```diff
@@ -1111,9 +1111,9 @@ if(SWIFT_INCLUDE_TOOLS)
   # Refer to the large comment above the add_subdirectory(stdlib) call.
   # https://bugs.swift.org/browse/SR-5975
   add_subdirectory(tools)
-endif()
 
-add_subdirectory(localization)
+  add_subdirectory(localization)
```

Comment:
@Ktwu I am fine with this. My only question is if this is the right place to put localization. But I am not going to litigate that here. Can you in a follow on commit add a large comment here explaining why this needs to be here. The reason why I am asking you to do this is that people do not touch CMakeLists.txt all the time so this historical context is often lost. Better to have it inline.

Reply:
Yeah, of course!

Reply:
I'm adding a comment here: https://github.com/apple/swift/pull/33086